packages/nitro-protocol/gas-benchmarks/vanillaSetup.ts

```diff
@@ -70,19 +70,32 @@ expect.extend({
     received: any, // TransactionResponse
     benchmark: number
   ) {
-    const {gasUsed} = await received.wait();
-    const pass = (gasUsed as BigNumber).eq(benchmark); // This could get replaced with a looser check with upper/lower bounds
+    const {gasUsed: gasUsedBN} = await received.wait();
+    const gasUsed = (gasUsedBN as BigNumber).toNumber();
+
+    const pass = gasUsed === benchmark; // This could get replaced with a looser check with upper/lower bounds
+
     if (pass) {
       return {
         message: () => `expected to NOT consume ${benchmark} gas, but did`,
         pass: true,
       };
     } else {
-      const diff: BigNumber = (gasUsed as BigNumber).sub(benchmark);
-      const diffStr: string = diff.gt(0) ? '+' + diff.toString() : diff.toString();
+      const format = (x: number) => {
+        return x.toLocaleString().replace(/,/g, '_');
+      };
+      const green = (x: string) => `\x1b[32m${x}\x1b[0m`;
+      const red = (x: string) => `\x1b[31m${x}\x1b[0m`;
```

Comment:
Did you consider https://www.npmjs.com/package/chalk ?

Reply:
I did, but I'm usually hesitant at importing things for one-liners. This may be irrational. Should I do it?

Reply:
Not necessarily. I think your rationale makes sense.